Sources/Winget-AutoUpdate/Winget-Upgrade.ps1

```diff
@@ -422,99 +355,81 @@ if (Test-Network)
         $outdated = Get-WingetOutdatedApps -src $Script:WingetSourceCustom;
 
         #If something unusual happened or no update found
-        if ($outdated -like "No update found.*")
-        {
+        if ($outdated -like "No update found.*") {
             Write-ToLog "$outdated" "cyan"
         }
         #Run only if $outdated is populated!
-        else
-        {
+        else {
             #Log list of app to update
-            foreach ($app in $outdated)
-            {
+            foreach ($app in $outdated) {
                 #List available updates
                 $Log = "-> Available update : $($app.Name). Current version : $($app.Version). Available version : $($app.AvailableVersion)."
-                $Log | Write-Host
+                $Log | Write-Output
```

Comment:
Fixes `Use Write-Output, Write-Verbose, or Write-Information instead of Write-Host` Linter Error